### PR TITLE
feat(YouTube - Searchbar): Restyle

### PR DESCRIPTION
**Pull request by Francesco146**
ref:

https://github.com/inotia00/revanced-integrations/pull/47

### CODE REVIEW

**Comment by ILoveOpenSourceApplications** [comment]
Can we have a screenshot of how it's going to look?

**Comment by Francesco146** [comment]
<details>
  <summary>Here's some screenshots</summary>

IMG_20240626_061744_979
IMG_20240626_061741_423
IMG_20240626_061736_668
IMG_20240626_061734_540


</details>

**Comment by Francesco146** [comment]
to mimic yt instead of "Search settings" as text hint, we could put:

"Search {customName}"

from the patch options. So the result will be this:

"Search ReVanced Extended"

or 

"Search FooBar" (if the user patched with Custom RVX Setting Name = FooBar

**Comment by ILoveOpenSourceApplications** [comment]
Do you think it's within the scope of this PR to resolve https://github.com/inotia00/ReVanced_Extended/issues/2130?

**Comment by Francesco146** [comment]
I'll work on that request, but I think it is better to divide the work into several steps. otherwise we wait more and more time without releasing anything new. to make the searchbar a different page, I need to study more how android development on views works. 

so no, this PR will only have the restyle, the custom name in the text hint and the code suggestions that inotia will propose

**Comment by Rairof** [comment]
https://github.com/inotia00/revanced-patches/pull/58#issuecomment-2190549746
@Francesco146 
it would be nice if you could consider adding the option to customize the pre-text inside the search bar (i.e "Search settings" in your screenshots)  within patch option to user's choice similar to how we can change RVX settings name through its patch if it is possible.
I know this sounds similar to your comment in https://github.com/inotia00/revanced-patches/pull/58#issuecomment-2190616338 but I was thinking on giving the users complete customization option on having whatever text in it rather then just being able to change `Search(CustomName)` in it.

**Comment by Francesco146** [comment]
immagine

done, note that the current translated strings should be updated to match their YouTube language (in order to include the `%s`)


---

@Rairof in a new PR maybe

File: src/main/resources/youtube/settings/layout/revanced_settings_with_toolbar.xml [hunk 1]
```diff
@@ -33,7 +33,9 @@
             android:searchIcon="@null"
             android:queryHint="@string/revanced_extended_settings_search_title"
             android:layout_margin="5dp"
-            android:padding="5dp" />
+			android:queryBackground="@android:color/transparent"
+			app:queryBackground="@android:color/transparent"
```
Comment (inotia00) [inline src/main/resources/youtube/settings/layout/revanced_settings_with_toolbar.xml:37]: properties are duplicated

File: src/main/resources/youtube/settings/layout/revanced_settings_with_toolbar.xml [hunk 2]
```diff
@@ -33,7 +33,9 @@
             android:searchIcon="@null"
             android:queryHint="@string/revanced_extended_settings_search_title"
```
Comment (inotia00) [inline src/main/resources/youtube/settings/layout/revanced_settings_with_toolbar.xml:34]: queryHint is overridden in integrations

**Comment by inotia00** [comment]
Since I am not granted access to the PR branch, I will make changes manually after the merge

**Pull request merged**